aep/general/0111/aep.md

```diff
@@ -0,0 +1,66 @@
+# Planes
+
+Resources and methods on an API can be divided into the _plane_ that they
+reside or perform operations upon. For the context of APIs, the following
+planes are defined:
+
+- Management plane: a uniform, resource-oriented API that primarily configures
+  and allows retrieval of resources.
+- Data plane: a heterogenous API (ideally resource-oriented) that reads and
+  write user data. Often connects to entities provisioned by the management
+  plane, such as virtual machines.
+
```

Comment:
I made a bunch of local suggestions above, but on the whole, I'm confused by the point this is trying to make.  The management plane reads and configures resources; the data plan is for reading and writing user data (also "ideally" in the form of resources).

I'm not sure that someone with no context could easily categorize APIs into the right plane given these definitions.

Reply:
...I'm not even sure that I can.  Would creating a Facebook group be a management plane or data plane method?  The group name/description/etc. are UGC, but if individual posts in the group are part of the data plane then does that place group management in the management plane?

If these resources are all in the same API, and APIs as a whole are categorized as data plane or management plane, does that mean this API is in the data plane?

What if `CreateBlobStore` and `CreateBlob` are in the same API?  Why wouldn't they be?

Reply:
I think on the whole we probably should discuss these in-person - would be a lot easier.

But answering your questions:

Facebook group would be management plane. Planes are somewhat contextual, but you create a facebook group and users post in them. 

> If these resources are all in the same API, and APIs as a whole are categorized as data plane or management plane, does that mean this API is in the data plane?

I don't think APIs have to be categorized as completely data / management. I think this could be done on the *operation* level. WDYT? I can update the doc.

> What if CreateBlobStore and CreateBlob are in the same API? Why wouldn't they be?

Yep that's fine, see above.